app/javascript/react/screens/App/Overview/components/InfrastructureMappingsList/InfrastructureMappingsList.js

```diff
@@ -1,226 +1,574 @@
+                      mapping.transformation_mapping_items,
+                      'storage'
+                    );
+                    const sourceLanCount = this.sourceCount(
+                      mapping.transformation_mapping_items,
+                      'lan'
+                    );
+                    const targetLanCount = this.destinationCount(
+                      mapping.transformation_mapping_items,
+                      'lan'
+                    );
+                    const associatedPlansCount =
+                      mapping.service_templates &&
+                      mapping.service_templates.length;
+
+                    const {
+                      targetClusters,
+                      targetDatastores,
+                      targetNetworks
+                    } = mapInfrastructureMappings(
+                      mapping.transformation_mapping_items,
+                      clusters,
+                      datastores,
+                      networks
+                    );
+
+                    return (
+                      <ListView.Item
+                        key={mapping.id}
+                        heading={mapping.name}
+                        description={
+                          <small>
+                            {__('Completed: ')}
+                            {formatDateTime(mapping.created_at)}
+                          </small>
+                        }
+                        stacked
+                        compoundExpand
+                        compoundExpanded={mapping.expanded}
+                        onCloseCompoundExpand={() => this.closeExpand(mapping)}
+                        additionalInfo={[
+                          <ListView.InfoItem key={0}>
+                            <ListView.Expand
+                              expanded={
+                                mapping.expanded && mapping.expandType === 0
+                              }
+                              toggleExpanded={() => {
+                                this.toggleExpand(mapping, 0);
+                              }}
+                            >
+                              <Icon type="pf" name="network" />
+                              <div className="mappings-expand-label-group">
+                                <div className="mappings-expand-label">
+                                  <strong>{sourceLanCount}</strong>
+                                  {` `}
+                                  {simplePluralize(
+                                    sourceLanCount,
+                                    __('Source Network'),
+                                    __('Source Networks')
+                                  )}
+                                </div>
+                                <div className="mappings-expand-label">
+                                  <strong>{targetLanCount}</strong>
+                                  {` `}
+                                  {simplePluralize(
```

Comment:
Still problematic, for mostly the same reasons .. (or see the list of plural rules in https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals )

Reply:
Ok - so this is becoming a PF design issue as well I'm afraid. If we need to style the count separately from the label, then we'd need to extract two strings. I think your original suggestion breaks this design. I will follow up on this...

Reply:
Another option may be to render that count alongside the text and just hide the count in the case we are displaying the label, however I think this would also be confusing to a translator. Has this not come up before? Displaying counts with differing styles?

Reply:
Will just remove the count styling for now.

Reply:
@himdel From a Gaprindashvili standpoint, I don't think we can do this (what you suggested above) -
```js
n_(`${sourceLanCount} Source Network`, `${sourceLanCount} Source Networks`)
```
If the support for enhanced pluralization has been added in ui-classic, it must be pretty new.

@priley86 We can look into enhanced pluralization all throughout the app after the release is out the door.

Reply:
@AparnaKarve Not really, it was introduced together with the rest, back in December 2015.